app/src/main/java/com/kodeco/android/coordplot/HorizontalSlider.kt

```diff
@@ -0,0 +1,15 @@
+package com.kodeco.android.coordplot
+
+import androidx.compose.material.Slider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun createHorizontalSlider(modifier: Modifier = Modifier, value : Float =0.5f, changeValue : (Float) -> Unit){
+    Slider(
+        value = value,
+        valueRange = 0.01f..1f,
+        onValueChange = changeValue
+
+    )
+}
```

Comment:
Note the EOF symbol here.  

<img width="36" alt="Screen Shot 2023-09-05 at 8 54 01 PM" src="https://github.com/sgrem/CoordinatePlot/assets/2581931/00c51950-0559-46a3-ba69-0a5fe3e8b4ca">

Typically all files need an extra newline at the end.  You can set this up in Android Studio via Preferences > Editor > General and in the On Save section ensure that the `Ensure every saved file ends with a line break` is checked.

<img width="1094" alt="Screen Shot 2023-09-05 at 8 54 58 PM" src="https://github.com/sgrem/CoordinatePlot/assets/2581931/22e86e3d-10ce-4727-a085-0e540e4e2106">